### ISSUE TITLE
Dangerous type piracy

### ISSUE THREAD

**Issue by ChrisRackauckas** [ThummeTo/FMISensitivity.jl#2]
https://github.com/ThummeTo/FMISensitivity.jl/blob/main/src/FMISensitivity.jl#L40-L42 that's not great.

**Comment by ThummeTo** [comment]
Yep, you got me :) 
Is there a reason why `increment_deriv!` is not defined for a `ZeroTangent`-increase?
In my head incrementing a TrackedReal by ZeroTangent shoud be defined, but just be a no-op.

If not, I need to dig deeper and prevent this call from happening...

**Comment by ChrisRackauckas** [comment]
If this is needed then upstream it?

ZeroTangent though is from ChainRules.jl, which ReverseDiff doesn't integrate with. This seems like however you did some chainrules thing forgot to handle this case.

**Comment by ThummeTo** [comment]
you are right, I will check for that... maybe the @grad_from_chainrules macro

**Comment by ThummeTo** [comment]
you are right, I will check for that... maybe the @grad_from_chainrules macro

**Comment by ThummeTo** [comment]
fixed in v0.3.1
